app/main/views/briefs.py

```diff
@@ -203,3 +203,39 @@ def delete_draft_brief(brief_id):
         abort(400, "Database Error: {0}".format(e))
 
     return jsonify(message="done"), 200
+
+
+@main.route("/briefs/<int:brief_id>/clarification-questions", methods=["POST"])
+def add_clarification_question(brief_id):
+    updater_json = validate_and_return_updater_request()
+
+    json_payload = get_json_from_request()
+    json_has_required_keys(json_payload, ['clarificationQuestion'])
+    question_json = json_payload['clarificationQuestion']
+    json_has_required_keys(question_json, ['question', 'answer'])
+
+    brief = Brief.query.filter(
+        Brief.id == brief_id
+    ).first_or_404()
+
+    question = brief.add_clarification_question(
```

Comment:
Good call, I've added some validation to the `BriefClarificationQuestion` model.